docs/helpers.rst

```diff
@@ -168,7 +168,23 @@ To use `client` as an authenticated standard user, call its
         response = client.get('/private')
         assert response.content == 'Protected Area'
 
-.. fixture:: admin_client
```

Comment:
This line should remain, right above the `admin_client` title.

And above the `async_client` title, should add

```
.. fixture:: async_client
```